open_seq2seq/encoders/tdnn_encoder.py

```diff
@@ -95,6 +98,19 @@ def _encode(self, input_dict):
     """
 
     source_sequence, src_length = input_dict['source_tensors']
+    
+    num_pad = tf.constant(0)
+
+    if isinstance(self._model.get_data_layer(), Speech2TextDataLayer):
```

Comment:
We should do an `if backend is librosa` check here.

Reply:
Fixed